### DISCUSSION TITLE
How do i make emails appear in List Of emails widget

### DISCUSSION THREAD

**Discussion by rohanjhanepal** [YetiForceCompany/YetiForceCRM#13719] [How to]
I am trying to make all the sent emails related to a vendor appear in the widget (List Of Emails ). But it doesn't appear.
Below is screen shot where there is a widget called 'Emails' where all emails sent should appear.

![Screenshot (53)](https://user-images.githubusercontent.com/40380541/97856765-dab11a80-1d24-11eb-9831-8f3c800ae08d.png)

yetiforce 5.3.0
Smtp and roundcube mail is configured and working well within the system.

**Comment by apcloic** [comment]
@rohanjhanepal 
Hi, you have to configure Mail Scanner :
https://gitstable.yetiforce.com/index.php?module=OSSMailScanner&parent=Settings&view=Index&block=8&fieldid=44
#13514
